docs/changelog.txt

```diff
@@ -44,6 +44,7 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 - `search`: fixed an issue causing item counts on the trade screen to display inconsistently when searching
 - `stockpiles`: fixed a crash when loading food stockpiles
 - `stockpiles`: fixed an error when saving furniture stockpiles
+- `embark-assistant`: Fixed issue causing incursion resource matching to skip the checks if those resources were provided only through incursions.
```

Comment:
"The checks" are the complete set of embark matching ones. The problem here was that the high level preliminary checks excluded world tiles that shouldn't have been excluded. Checks (or matching) are performed in two steps; first a world tile match to skip world tile that can't have any matches (which is where the logic failed), and then a detailed check of each possible embark within the remaining world tiles.

It's odd that my work is based on an older version as I thought I did get the latest before I started (earlier the same day), and getting those things cause a whole lot of stuff to be changed, so I have no idea why it wouldn't be based on the latest data.

Reply:
Got it, thanks.

Hm, maybe you pulled from the `master` branch? At least in this repo, that points to the latest stable release, and `develop` has changes made since then.

For reference, if it helps:
* 0.47.04-r3 history: https://github.com/DFHack/dfhack/commits/0.47.04-r3
* This PR's history: https://github.com/DFHack/dfhack/commits/0ef56ba859c2328957228385112da5f42b44d074 (basically, your commits + 0.47.04-r3)
* `develop` history: https://github.com/DFHack/dfhack/commits/develop

It'll merge successfully now, in any case - I'll just fix the changelog manually. Basing work on `develop` would help avoid merge conflicts, but in this case, it looks like no conflicting changes to embark-assistant have been made since r3. (I realize other DFHack repos like df-structures use `master` for everything - perhaps we should make them more consistent...)